File: 백준/Bronze/2908. 상수/상수.cc
#include <stdio.h>

int main() {
    char first[4], second[4];
    scanf("%s %s", first, second);

    int reversed_first = (first[2] - '0') * 100 + (first[1] - '0') * 10 + (first[0] - '0');
    int reversed_second = (second[2] - '0') * 100 + (second[1] - '0') * 10 + (second[0] - '0');

    if (reversed_first > reversed_second) {
        printf("%d", reversed_first);
    } else {
        printf("%d", reversed_second);
    }

    return 0;
}

File: 백준/Bronze/2908. 상수/상수.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String first = scanner.next();
        String second = scanner.next();

        int reversedFirst = (first.charAt(2) - '0') * 100 + (first.charAt(1) - '0') * 10 + (first.charAt(0) - '0');
        int reversedSecond = (second.charAt(2) - '0') * 100 + (second.charAt(1) - '0') * 10 + (second.charAt(0) - '0');

        if (reversedFirst > reversedSecond) {
            System.out.println(reversedFirst);
        } else {
            System.out.println(reversedSecond);
        }
    }
}
